docs/source/installation.rst

```diff
@@ -48,46 +49,132 @@ You're done! Now go to:
 .. _docker: https://www.docker.io/
 
 
-MozDef manual installation process on RedHat systems
-----------------------------------------------------
+Manual Docker Installation
+--------------------------
 
-Summary
-*******
-This section explains the manual installation process for the MozDef system.
-  git clone https://github.com/mozilla/MozDef.git
+*Use sudo whereever required*
 
+**(Instructions for apt-based systems using Docker)**
 
 
-Elasticsearch nodes
--------------------
+1. Cloning repository ::
 
-This section explains the manual installation process for Elasticsearch nodes (search and storage).
+    $ export MOZDEF_PATH=/opt/MozDef
+    $ git clone https://github.com/mozilla/MozDef.git $MOZDEF_PATH
 
-ElasticSearch
-*************
+2. Installing dependencies ::
 
-Installation instructions are available on `Elasticsearch website`_.
-You should prefer packages over archives if one is available for your distribution.
+    # RabbitMQ
+    $ apt-get install -y rabbitmq-server
+    $ rabbitmq-plugins enable rabbitmq_management
 
-.. _Elasticsearch website: http://www.elasticsearch.org/overview/elkdownloads/
+    # MongoDB
+    $ apt-get install -y mongodb
 
-Marvel plugin
-*************
+    # NodeJS and NPM
+    $ curl -sL https://deb.nodesource.com/setup_0.12 | sudo bash -
+    $ apt-get install -y nodejs npm
 
-`Marvel`_ is a monitoring plugin developed by Elasticsearch (the company).
+    # Nginx
+    $ apt-get install -y nginx-full
+    $ cp $MOZDEF_PATH/docker/conf/nginx.conf /etc/nginx/nginx.conf
 
-WARNING: this plugin is NOT open source. At the time of writing, Marvel is free for 30 days.
-After which you can apply for a free basic license to continue using it for it's key monitoring features.
+    # Libraries
+    $ apt-get install -y python2.7-dev python-pip curl supervisor wget libmysqlclient-dev
+    $ pip install -U pip
 
-To install Marvel, on each of your elasticsearch node, from the Elasticsearch home directory::
+3. Installing python libraries ::
 
-  sudo bin/plugin install license
-  sudo bin/plugin install marvel-agent
-  sudo service elasticsearch restart
+    $ pip install uwsgi celery virtualenv
```

Comment:
uwsgi and celery are part of requirements.txt. so you shouldn't need to explicitly pip install them.